x-pack/legacy/plugins/apm/public/components/app/Settings/CustomizeUI/CustomActions/EmptyPrompt.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { EuiButton, EuiEmptyPrompt } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+
+export const EmptyPrompt = ({
+  setIsFlyoutOpen
+}: {
+  setIsFlyoutOpen: (isFlyoutOpen: boolean) => void;
+}) => {
+  return (
+    <EuiEmptyPrompt
+      iconType="boxesHorizontal"
+      iconColor=""
+      title={
+        <h2>
+          {i18n.translate(
+            'xpack.apm.settings.customizeUI.customActions.emptyPromptTitle',
+            {
+              defaultMessage: 'No actions found.'
+            }
+          )}
+        </h2>
+      }
+      body={
+        <>
+          <p>
+            {i18n.translate(
+              'xpack.apm.settings.customizeUI.customActions.emptyPromptText',
+              {
+                defaultMessage:
+                  "Let's change that! You can add custom actions to the Actions context menu by the trace and error details for each service. This could be linking to a Kibana dashboard or go to your organization's support portal."
```

Comment:
I think this reads better with consistent participles:

> This could be _linking_ to a Kibana dashboard or _going_ to your organization's support portal

vs. 

> This could be _linking_ to a Kibana dashboard or _go_ to your organization's support portal